test/integration.test.js

```diff
@@ -22,11 +23,20 @@ describe('css-vendor', () => {
       expect(supportedProperty('display')).to.be('display')
     })
 
-    it('should prefix if needed', () => {
-      const prop = supportedProperty(`${prefix.css}animation`)
-      if (prop !== 'animation') {
-        expect(prop).to.be(`${prefix.css}animation`)
-      }
+    browser("requires prefix for transform property", {
```

Comment:
Why is this a hack? I used the values from caniuse as they are static and won't ever change, there is no need to query the api and introduce another dependency that could break.